validator/cardTypes.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
This should not be a `.json` file. Build tools such as webpack require a special JSON loader when `.json` files are used, making this a normal commonjs file will work just as well and not require any special behavior in build tools.

Reply:
Good point, changed to a js file